src/mir/construction.md

```diff
@@ -1 +1,111 @@
 # MIR construction
+
+The lowering of [HIR] to [MIR] occurs for the following (probably incomplete)
+list of items:
+
+* Function and Closure bodies
+* Initializers of `static` and `const` items
+* Initializers of enum discriminants
+* Glue and Shims of any kind
+    * Tuple struct initializer functions
+    * Drop code (the `Drop::drop` function is not called directly)
+    * Drop implementations of types without an explicit `Drop` implementation
+
+The lowering is triggered by calling the `mir_built` query. The entire lowering
+code lives in `src/librustc_mir/build`. There is an intermediate representation
```

Comment:
It's best to link to the rustdoc for whatever you reference, in my opinion. That way the linkchecker can tell us when the lints are wrong. Here, I would probably link to the provider for the `mir_built` query at <https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir/transform/fn.mir_built.html>.